src/components/messageSubmitInterface/inputField/sendButton.styles.scss

```
@@ -16,7 +17,8 @@
 	appearance: none;
 	cursor: pointer;
 	overflow: hidden;
-	background-color: var(--m3-primary-fixed, #ffdad5);
+	// #597: resting / not-selected composer — #FFB4AA
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)